src/client/lazy-app/Compress/index.tsx

```diff
@@ -374,6 +376,15 @@ export default class Compress extends Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props, prevState: State): void {
+    const isLoading =
+      this.state.loading ||
+      this.state.sides[0].loading ||
+      this.state.sides[1].loading;
+    if (isLoading) {
```

Comment:
alternatively, i could do this, but it seemed unnecessary:
```suggestion
    const wasLoading = prevState.loading;
    const isLoading = this.state.loading || this.state.sides[0].loading || this.state.sides[1].loading;
    const changedLoadingState = wasLoading !== isLoading;
    if (changedLoadingState) {
```

Reply:
This seems better to me, as it avoids going though the title setter if the title doesn't need to change.